pkg/git/git.go

```diff
@@ -0,0 +1,70 @@
+package git
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/codefresh-io/cli-v2/pkg/log"
+)
+
+
+var (
+	requiredGitHubScopes = []string{ "repo", "admin:repo_hook" }
+)
+
+
+func VerifyToken(ctx context.Context, provider string, token string) (bool, error) {
+	providerToVerifier := map[string]func(context.Context, string)(bool, error){}
+
+	providerToVerifier["github"] = verifyGitHubTokenScope
+	providerToVerifier["gitlab"] = verifyGitLabTokenScope
+
+	verifier := providerToVerifier[provider]
+
+	return verifier(ctx, token)
+}
+
+func verifyGitHubTokenScope(ctx context.Context, token string) (bool, error) {
+	log.G(ctx).Info("Verifing your git token")
```

Comment:
This log can be outside, possibly in `runtime.go`

Reply:
i thought so too, but it will look weird in case of gitlab provider

Reply:
why?

Reply:
so maybe just remove it, library code should generally not print info logs.